Jenkinsfile.github

```diff
@@ -24,7 +24,7 @@
  *
  */
 @Library('cms-meta-tools') _
-@Library('csm-shared-library') __
+@Library('csm-shared-library@feature/deprecate-multiarch') __
```

Comment:
This change needs to be rolled back after https://github.com/Cray-HPE/csm-shared-library/pull/52 is merged.